modules/sdk-coin-eth/src/lib/utils.ts

```diff
@@ -61,8 +64,12 @@ import {
   walletInitializationFirstBytes,
   v1CreateForwarderMethodId,
   walletSimpleConstructor,
+<<<<<<< HEAD
   createWalletTypes,
   v1CreateWalletMethodId,
+=======
+  createV1ForwarderTypes,
+>>>>>>> 27bae27d3 (fix(sdk-coin-eth): fix v1 address init code)
```

Comment:
Merge conflict comments need to be removed here.

Reply:
working on it

Reply:
done